### PR TITLE
Documentation changes post validation

### PR DESCRIPTION
**Pull request by kothariaman**
### What does this Pull Request accomplish?
Updated comments and message description post feedback given by team after demo.

### Why should this Pull Request be merged?
Comments changes for better meaningful information.

### What testing has been done?
Locally tested the python version.

### CODE REVIEW

File: examples/nixnet/lin-signal-single-point-output.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
  This is used to demonstrate a signal single point output session. 
  This example uses hardcoded signal names that use the NIXNET_example database.
```
Comment (doshirohan) [inline examples/nixnet/lin-signal-single-point-output.py:5]: This example uses `NIXNET_exampleLDF` database, not `NIXNET_example`. Looks like these comments were copied from one python example to another. There might be slight changes based on device, databases and mode you are using in each example. Compare comments from all example with equivalent C examples.

File: examples/nixnet/lin-signal-single-point-output.py [hunk 2]
```diff
@@ -4,6 +4,8 @@
  This is used to demonstrate a signal single point output session. 
  This example uses hardcoded signal names that use the NIXNET_example database. 
  Also ensure that the transceivers are externally powered when using C Series modules.
```
Comment (doshirohan) [inline examples/nixnet/lin-signal-single-point-output.py:6]: Same comment about "Also ensure...". It should be after you talk about database editor.

**Review by harsha-bhushan — APPROVED** [review]

File: examples/nixnet/flexray-signal-single-point-input.py [hunk 3]
```diff
@@ -4,6 +4,8 @@
  This is used to demonstrate a signal single point input session. 
  This example uses hardcoded signal names that use the NIXNET_example database. 
  Also ensure that the transceivers are externally powered when using C Series modules.
```
[the remark this answers is not in the document]
Comment (kothariaman) [reply examples/nixnet/flexray-signal-single-point-input.py:6]: Done.

File: examples/nixnet/lin-signal-single-point-output.py (hunk 2 above)
Comment (kothariaman) [reply examples/nixnet/lin-signal-single-point-output.py]: Done

File: examples/nixnet/ethernet-frame-stream-input-reader.py [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
   This example reads all the frames on the network and displays them. 
   This is used to demonstrate a frame input stream session.
+  To use your own database, you need to add an alias to your database file using the NI-XNET
```
[the remark this answers is not in the document]
Comment (kothariaman) [reply examples/nixnet/ethernet-frame-stream-input-reader.py]: Done.

File: examples/nixnet/lin-signal-single-point-output.py (hunk 1 above)
Comment (kothariaman) [reply examples/nixnet/lin-signal-single-point-output.py]: Done.

**Review by astarche — APPROVED** [review]

**Pull request merged**